redfish-core/lib/health.hpp

```diff
@@ -126,7 +126,6 @@ struct HealthPopulate : std::enable_shared_from_this<HealthPopulate>
             if (boost::starts_with(path.str, globalInventoryPath) &&
                 boost::ends_with(path.str, "critical"))
             {
-                health = "Critical";
```

Comment:
Should you also remove health everywhere in this file?

Reply:
I don't think so. Other resources also use this file. They need to set health here.
https://github.com/ibm-openbmc/bmcweb/search?q=health-%3Epopulate

Reply:
I thought we said "health use operationalStatus" "healthrollup, use this design" ?

Reply:
> I thought we said "health use operationalStatus" "healthrollup, use this design" ?

For powersupply, fan, CPU and DIMM, it is right now.
But there are some old codes that still use the original design. For example, the health of storage, task and Ethernet. Do you want me to change all these old code into this design?

Reply:
The problem with this is sometimes we have health being set by both this function and the operational status in the route
E.g.
https://github.com/openbmc/bmcweb/blob/11a2f0f0186983d5939351c276c60b054c059c5a/redfish-core/lib/memory.hpp#L450
https://github.com/openbmc/bmcweb/blob/11a2f0f0186983d5939351c276c60b054c059c5a/redfish-core/lib/memory.hpp#L482

Might need to do something better later here